gpytorch/kernels/rbf_kernel.py

```diff
@@ -89,8 +90,31 @@ def __init__(
             eps=eps,
         )
 
-    def forward(self, x1, x2, **params):
+    @torch.jit.script
```

Comment:
This is quite repetitive - why not define `__jit_sq_dist` as a utility function outside the `Kernel` class, then you should be able to have the  `__jit_rbf_kernel` just call that.

Reply:
For efficiency -- I'm not sure that if we call a scripted function (or even whether we *can*) within a scripted function whether the JIT will compile the whole trace or not. The whole point of defining the custom function for RBF is so that the exponentiation happens in the JIT script, which seems to save us a kernel matrix copy worth of memory.